### ISSUE TITLE
hello sir how can I convert your train from pytorch lighting to pytorch 

### ISSUE THREAD

**Issue by Ahme1994** [bdsaglam/torch-scae#2]
hello sir how can I convert your train from pytorch lighting to pytorch

**Comment by Ahme1994** [comment]
I have special datasets 4 option label test , label train , train dataset ,test dataset 
how can I make it four

**Comment by bdsaglam** [comment]
Hi, I would advise to use PyTorch Lightning trainer since everything is already set up. You can create a Lightning module similar to the one provided for MNIST by customizing the methods for your dataset. 
To train without Lightning, take a look at train step of SCAEMNIST class.

**Comment by Ahme1994** [comment]
thank you sir i will trying

**Comment by Ahme1994** [comment]
hello sir when I was trying about what you said   i do not get it 
that is my datasets 
how can I convert it by pytorch lighting 
#Randomly take two numbers
for i in ['13','25','30']:
    train_data = dic_train[i]
    test_data = dic_test[i]
    train_label = dic_train_label[i]
    test_label = dic_test_label[i]
    #Make loader
    train_dataset = Data.TensorDataset(train_data,train_label)
    test_dataset = Data.TensorDataset(test_data,test_label)
    train_loader = Data.DataLoader(dataset=train_dataset,batch_size=batchsize,shuffle=True)
    test_loader = Data.DataLoader(dataset=test_dataset,batch_size=batchsize,shuffle=True)
 thanks  advance

**Comment by bdsaglam** [comment]
This model is for vision tasks so it does not work for such data. I would
recommend you to check PyTorch tutorials.
https://pytorch.org/tutorials/
